src/DataBuilder.php

```diff
@@ -980,4 +1007,27 @@ public function needsTruncating(array $payload)
     {
         return strlen(json_encode($payload)) > self::MAX_PAYLOAD_SIZE;
     }
+    
+    /**
+     * Wrap a PHP error in an ErrorWrapper class and add backtrace information
+     *
+     * @param string $errno
+     * @param string $errstr
+     * @param string $errfile
+     * @param string $errline
+     *
+     * @return ErrorWrapper
+     */
+    public function generateErrorWrapper($errno, $errstr, $errfile, $errline)
+    {
+        if ($this->captureErrorStacktraces) {
+            $backTrace = array_slice(
+                debug_backtrace($this->localVarsDump ? 0 : DEBUG_BACKTRACE_IGNORE_ARGS),
+                2
```

Comment:
Is this still 2 or should it be 3? I am not sure if we added a level of indirection here.